docs/extend/extend-sdks.md

```diff
@@ -5,5 +5,6 @@ The Zowe SDKs are open source. You can contribute to add features, enhancements,
 The functionality is currently limited to the interfaces provided by IBM z/OSMF. As a plug-in developer, you can enhance the SDK by creating a packages that exposes programmatic APIs for your service.
```

Comment:
Change:
As a plug-in developer, you can enhance the SDK by creating a packages that exposes programmatic APIs for your service.
to:
As a plug-in developer, you can enhance the SDK by creating a package that exposes programmatic APIs for your service.